### PR TITLE
reworked sponsor block

### LINKED ISSUE #561: Add Netlify Sponsorship
Netlify is sponsoring the AIID with more generous build system limits at zero price. They request the following,

```
...we would appreciate a link to our service on your project's main web page for your website's visitors to see (so, not just in your repository documentation or on a sponsors page). You have two options: we have pre-made badges (https://www.netlify.com/press/#badges) for your convenience, or you may create your own link, which should read “This site is powered by Netlify”. Either should link back to our home page, https://www.netlify.com/.
```

### PR DESCRIPTION
**Pull request by smcgregor**
fixes #561

This pull request reworks the sponsor and founding blocks into being one card and puts most of the details about the sponsorships into modals that can be accessed by clicking on the logos. The sponsor block is on both the landing page and the end of the `about` page.

Screen Shot 2022-06-27 at 11 50 12 PM

### CODE REVIEW

File: site/gatsby-site/src/components/landing/Sponsors.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+export default function Sponsors({ className }) {
+  const [modalState, setModalState] = useState('close');
+
+  return (
+    <Card className={className}>
+      <Card.Body>
+        <Card.Title as="h2">The Responsible AI Collaborative</Card.Title>
+        <Row className="g-0">
+          <Col lg={5} md={12} sm={12}>
+            <Card.Body>
```
Comment (lmcnulty) [inline site/gatsby-site/src/components/landing/Sponsors.js:82]: Having another `<Card.Body>` here makes the text not aligned with the header / cards above.



If you didn't intend for that to be the case, you could keep the gap between the columns by adding margin-left to the right `<Col>` when it's wide enough to not wrap.

**Review by lmcnulty — CHANGES_REQUESTED** [review]
See the inline comments. The only thing I'd say should definitely change is that there should be space between the text and the image in the modals. The subtitles for the sponsor logos also need attention, but it's a matter of taste what the appropriate choice is. Everything else is fairly unimportant.

File: site/gatsby-site/src/components/landing/Sponsors.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+          social benefits of the AIID to their Partners and other stakeholders and have an ongoing
+          interest in supporting its outcomes.
+        </SponsorModal>
+
+        <SponsorModal
+          setModalState={setModalState}
+          modalState={modalState}
+          modalName={'N'}
+          title={'Netlify'}
+          imagePath={'/netlify-dark.svg'}
```
[the remark this answers is not in the document]
Comment (smcgregor) [reply site/gatsby-site/src/components/landing/Sponsors.js:162]: Updated locally, will push shortly.

File: site/gatsby-site/src/components/landing/Sponsors.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+                .
+              </Card.Text>
+              <a href="https://docsend.com/view/a45p7mgh44nu8x7j" target="_blank" rel="noreferrer">
+                <StyledImageCover src="/images/reportcover.png" />
+              </a>
+            </Card.Body>
+          </Col>
+          <Col lg={7} md={12} sm={12}>
+            <Card.Body>
+              <StyledSubtitle top={'0px'} margin={'0px'}>
```
[the remark this answers is not in the document]
Comment (smcgregor) [reply site/gatsby-site/src/components/landing/Sponsors.js]: Centered.

**Pull request merged**